version/version.go

```diff
@@ -1,5 +1,3 @@
 package version
 
-var (
-	Version = "0.0.1"
-)
+var Version = "0.0.1"
```

Comment:
is this variable used in any of our generation? should we update it?

Reply:
yes, it is only used at https://github.com/openshift/elasticsearch-operator/blob/8a339b3f9d5e3b636f156c5839852c431839de4e/cmd/manager/main.go#L46

Do you want me to update it to some value?

Reply:
yes, it should match the version of our CSV I think (for now)